lib/jekyll-github-metadata/site_github_munger.rb

```diff
@@ -30,7 +30,7 @@ def github_namespace
         when nil
           drop
         when Hash
-          Jekyll::Utils.deep_merge_hashes(site.config["github"], drop)
+          Jekyll::Utils.deep_merge_hashes(drop, site.config["github"])
```

Comment:
Flipping this may have some unintended consequences (because people expect the opposite) so we should be prepared for upset users.